src/harness/sas_testcase.py

```diff
@@ -19,6 +19,9 @@
 import unittest
 import sas_interface
 import sas
+import signal
```

Comment:
I think signal is a part of Python Standard Library https://docs.python.org/2/library/signal.html